plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -998,6 +998,9 @@ private static function get_all_woocommerce_options_values() {
 			'hpos_transactions_level'               => get_option( 'woocommerce_db_transactions_isolation_level_for_custom_orders_table_data_sync' ),
 			'show_marketplace_suggestions'          => get_option( 'woocommerce_show_marketplace_suggestions' ),
 			'admin_install_timestamp'               => get_option( 'woocommerce_admin_install_timestamp' ),
+			'coming_soon'                           => get_option( 'woocommerce_coming_soon' ),
```

Comment:
I've followed this [Slack thread](https://a8c.slack.com/archives/C06D0919X0U/p1735045974240699) but I'm not so sure if "coming_soon" indicates it's a staging WooCommerce site? 🤔 
![image](https://github.com/user-attachments/assets/9e6760b3-4425-42c6-9222-173e9d9d305f)

Reply:
Also, based on the screenshot above, there is a option "Live", which I believe is the "Launch Your Store" feature?

Reply:
@EveChen , as far as I understand, the `Coming soon` option is used on the live site, but the site is still hidden while the owner is trying something out. 
`Coming soon` and `Live` modes are mutually exclusive (radio button) and sets the `woocommerce_coming_soon` options to `yes` or `no` respectively.

Reply:
Thanks for the explanation @menakas! One follow up question, do you know if we regard merchants who chose `Coming soon` option as "staging sites"? The reason why I'm asking is that Payments Pod is working on monetized GMV and we want to filter out "staging sites". If the above is true, then we can use this logging to filter out staging sites 😃.

Reply:
@EveChen yes, this PR is to address the same problem : to exclude test orders made in the `Coming Soon` mode. But they're not staging sites because this option can be toggled - y'day `Coming soon`, today `Live` and tomorrow `Coming soon` again. So, we need some order-specific data. 

IMHO, we shouldn't regard these sites as staging sites. But, staging sites would also have the same options and that's going to complicate things, I guess.

Reply:
Gotcha! Just like you and Alex mentioned, looks like this feature is not that reliable. But still good to know we have this toggle for merchants to use (though I guess some of merchants may also confused about this, just like me 😅 )